From 2e2dd08b3fdbb7e899eae8ee072b894e96ffa246 Mon Sep 17 00:00:00 2001
From: Adrien Destugues <pulkomandy@gmail.com>
Date: Sun, 26 Oct 2014 17:47:29 +0100
Subject: Don't reset LIBRARY_PATH.

Using a variable named LIBRARY_PATH is not a good idea on Haiku. Just
rename it.

diff --git a/configure.ac b/configure.ac
index d8fc51b..5def208 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2885,15 +2885,15 @@ AC_DEFINE_UNQUOTED(EXECUTABLE_PATH,"$DEFINE_EXECUTABLE_PATH",[Directory where ex
 AC_SUBST(EXECUTABLE_PATH)

 # Path to ImageMagick lib
-LIBRARY_PATH="${LIB_DIR}/${LIBRARY_RELATIVE_PATH}"
-DEFINE_LIBRARY_PATH="${LIB_DIR}/${LIBRARY_RELATIVE_PATH}/"
+XLIBRARY_PATH="${LIB_DIR}/${LIBRARY_RELATIVE_PATH}"
+DEFINE_XLIBRARY_PATH="${LIB_DIR}/${LIBRARY_RELATIVE_PATH}/"
 case "${build_os}" in
   mingw* )
-    DEFINE_LIBRARY_PATH=`$WinPathScript "$DEFINE_LIBRARY_PATH" 1`
+    DEFINE_XLIBRARY_PATH=`$WinPathScript "$DEFINE_XLIBRARY_PATH" 1`
     ;;
 esac
-AC_DEFINE_UNQUOTED(LIBRARY_PATH,"$DEFINE_LIBRARY_PATH",[Directory where architecture-dependent files live.])
-AC_SUBST(LIBRARY_PATH)
+AC_DEFINE_UNQUOTED(LIBRARY_PATH,"$DEFINE_XLIBRARY_PATH",[Directory where architecture-dependent files live.])
+AC_SUBST(XLIBRARY_PATH)

 #
 # path to local binaries
@@ -2910,7 +2910,7 @@ AC_SUBST(LIB_BIN_DIR)

 LIB_BIN_DIR_RELATIVE_PATH=$LIB_BIN_DIR
 AC_SUBST(LIB_BIN_DIR_RELATIVE_PATH)
-LIB_BIN_DIR_PATH="${LIBRARY_PATH}/${LIB_BIN_DIR_RELATIVE_PATH}"
+LIB_BIN_DIR_PATH="${XLIBRARY_PATH}/${LIB_BIN_DIR_RELATIVE_PATH}"
 AC_SUBST(LIB_BIN_DIR_PATH)

 #
@@ -2930,7 +2930,7 @@ AC_SUBST(MODULES_DIRNAME)

 MODULES_RELATIVE_PATH="${MODULES_DIRNAME}"
 AC_SUBST(MODULES_RELATIVE_PATH)
-MODULES_PATH="${LIBRARY_PATH}/${MODULES_RELATIVE_PATH}"
+MODULES_PATH="${XLIBRARY_PATH}/${MODULES_RELATIVE_PATH}"
 AC_SUBST(MODULES_PATH)

 #
@@ -3023,7 +3023,7 @@ AC_SUBST(SHAREARCH_DIRNAME)

 SHAREARCH_RELATIVE_PATH="${SHAREARCH_DIRNAME}"
 AC_SUBST(SHAREARCH_RELATIVE_PATH)
-SHAREARCH_PATH="${LIBRARY_PATH}/${SHAREARCH_RELATIVE_PATH}"
+SHAREARCH_PATH="${XLIBRARY_PATH}/${SHAREARCH_RELATIVE_PATH}"
 AC_SUBST(SHAREARCH_PATH)

 #
--
2.10.2

